modules/nw-udn-support-matrix-primary-secondary.adoc

```diff
@@ -0,0 +1,152 @@
+//module included in the following assembly:
+//
+// *networkking/multiple_networks/understanding-user-defined-networks.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="support-matrix-for-primary-secondary-networks_{context}"]
+= Support matrix for primary and secondary networks
+
+The `UserDefinedNetwork` and `NetworkAttachmentDefinition` custom resources (CRs) provide cluster administrators and users the ability to create customizable network configurations and define their own network topologies, ensure network isolation, manage IP addressing for workloads, and leverage advanced network features. A third CR, `ClusterUserDefinedNetwork`, is also available, which allows administrators the ability to create and define additional networks spanning multiple namespaces at the cluster level. Both user-defined networks and network attachment definitions can serve as both the primary and secondary network interface, and each support `layer2` and `layer3` topologies; a third network topology, Localnet, is also supported with network attachment definitions with secondary networks. 
+
+[NOTE]
+====
+As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` leveraging primary networks and `ClusterUserDefinedNetwork` APIs. It is available for use with network attachment definitions that leverage secondary networks only.
```

Comment:
First sentence is a bit unclear, I am not sure what to suggest to clarify
```suggestion
As of {product-title} 4.18, the Localnet topology is unavailable for use with the `UserDefinedNetwork` leveraging primary networks and `ClusterUserDefinedNetwork` APIs. It is available for use with network attachment definitions that leverage secondary networks only.
```

Reply:
Fixed.